core/src/providers/auth/openIdConnect.js

```diff
@@ -162,12 +162,12 @@ export class openIdConnect {
             } else {
               window.location.href = window.location.origin;
             }
-            // resolve(true); // not resolving in order to not interrupt window.location.href
           }, 50);
         })
         .catch(err => {
           console.error(err);
-          reject(err);
+          localStorage.clear();
```

Comment:
- Is it not too harsch to clear the whole local storage? It would be more polite to just clear keys this auth provider handles.
- Do we clear auth data in non-error logout flows as well?
- Is there a reason to not use session storage for the auth data?

Reply:
Yes, you are right. We usually use `localStorage.removeItem('luigi.auth');` upon errors and normal logout. I guess localStorage is used to be able to close the window and still be logged in without going trough sso flow.

Reply:
Please doublecheck with Philipp if he is fine with using localStorae here, thanks!